refactor linux mixins to remove target option for run command (#1222)
beeware/briefcase: changes/1207.bugfix.rst

> Refactored Linux system mixins to remove ``target`` option and related processing from ``run`` command.

Reviewer: ```suggestion
The option to run Linux system packages through Docker was removed.
```
